tool/builder.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:async';
+
+import 'package:build/build.dart';
+import 'package:glob/glob.dart';
+import 'package:path/path.dart' as pathLib;
+
+String _resourcesFile(Iterable<String> packagePaths) => '''
+// WARNING: This file is auto-generated. Do not taunt.
+
+library dartdoc.html.resources;
```

Comment:
This is pre-refactor, but we likely don't need to generate a library directory for the resource file anymore.